hosts/cerberus.nix

```diff
@@ -0,0 +1,21 @@
+{ profiles, ... }:
+{
+  imports = with profiles; [
+    hetzner-cloud
+    base
+    nomad.client
+  ];
+
+  services.nomad.settings.client.host_volume = {
+    "vault/data" = {
+      path = "/var/lib/vault/data";
+    };
+  };
+
+  systemd.tmpfiles.rules = [
+    "d /var/lib/vault/data 0750 1000 1000"
+  ];
+
+  # Change this if you want to lose all data on this machine!
+  system.stateVersion = "23.11";
```

Comment:
This can only be dangerous to change on a machine that currently has persistent data on it, but since this server not yet exists it should start with stateVersion as new as possible